lib/store_base_sti_class_for_5_0.rb

```diff
@@ -346,6 +346,24 @@ def build_through_record(record)
     end
 
     module Reflection
+      class PolymorphicReflection < ThroughReflection
```

Comment:
Hi @januszm! Thanks for your review. Actually the class inherits from `AbstractReflection` in the [master branch](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/reflection.rb#L978) (which I think will be in Rails 5.1) and from `ThroughReflection` in the [5.0-stable branch](https://github.com/rails/rails/blob/5-0-stable/activerecord/lib/active_record/reflection.rb#L996) but I'll do as you said ;)